.drone.yml

```
@@ -389,16 +389,11 @@ steps:
       - git fetch --tags --force
 
   - name: build
```

```suggestion
   - name: static
```
There are steps below that depend on this being the step name